src/EVM/Solidity.hs

```diff
@@ -472,14 +473,14 @@ readStdJSON json = do
                            (HMap.filter (isJust . preview (key "content")) srcs)
         abis = force ("abi key not found in " <> show x) $
           toList <$> x ^? key "abi" % _Array
-      in (s <> ":" <> c, (SolcContract {
+      in (contractName, (SolcContract {
         runtimeCode      = theRuntimeCode,
         creationCode     = theCreationCode,
         runtimeCodehash  = keccak' (stripBytecodeMetadata theRuntimeCode),
         creationCodehash = keccak' (stripBytecodeMetadata theCreationCode),
         runtimeSrcmap    = force "srcmap-runtime" (makeSrcMaps (runtime ^?! key "sourceMap" % _String)),
         creationSrcmap   = force "srcmap" (makeSrcMaps (creation ^?! key "sourceMap" % _String)),
-        contractName = s <> ":" <> c,
+        contractName = contractName,
```

Comment:
```suggestion
```

Reply:
hmm, not sure I understood this one much, I was trying to preserve original code content.

Reply:
oh but let me change it anyhow, it might be ambiguous

Reply:
check again